### ISSUE TITLE
LAN browser duplicates server entries

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#764]
Hi, the built-in LAN server browser duplicates entries:

![duplicated LAN entry](http://dl.illwieckz.net/u/thomas-debesse/bazar/unvanquished/bugs/unvanquished_2015-07-07_195552_000.png)

**Comment by Amanieu** [comment]
I believe those are due to both IPv4 and IPv6 versions of the server being reported on the list.

**Comment by illwieckz** [comment]
Ah, it's probably that, yes! :-)

**Comment by illwieckz** [comment]
So, is this considered as a bug or not?
It could be good if the server sends a public unique value (like a GUID) to the client so the client can hide the IPV4 adress if an IPV6 adress for the same server is found.

**Comment by DolceTriade** [comment]
Yes, it is a bug, albeit a low priority one. A GUID would be a good idea. Perhaps @Veyrdite can consider this in his master server upgrade.

**Comment by Veyrdite** [comment]
NB the master server is not involved in LAN listings
I don't think I want to go the route of IDs if I can avoid it but I won't make any suggestions on what's possible until I have a better look at the client code.

**Comment by DolceTriade** [comment]
Fair enough, but I think this problem exists for regular servers too.

**Comment by Veyrdite** [comment]
I'm not sure how all of our IPV6 support will pan out.  At the moment the client tells the master if it wants ipv6 servers to be listed or not but I don't know if the client actually detects what it can use or if it's hard-coded to request both types.
